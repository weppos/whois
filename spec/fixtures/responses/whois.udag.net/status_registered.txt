
# Whois Server Version 1.49

Domain Name: udag.net
Registry Domain ID: 59973274_DOMAIN_NET-VRSN
Registrar WHOIS Server: whois.udag.net
Registrar URL: http://www.united-domains.de/
Updated Date: 2013-02-12T08:40:19Z
Creation Date: 2001-02-11T20:15:57Z
Registrar Registration Expiration Date: 2014-02-11T20:15:57Z
Registrar: united domains AG
Registrar IANA ID: 1408
Registrar Abuse Contact Email: abuse@united-domains.de
Registrar Abuse Contact Phone: +49.8151368670
Domain Status: clientTransferProhibited
Registry Registrant ID:
Registrant Name: Host Master
Registrant Organization: united-domains AG
Registrant Street: Gautinger Str. 10
Registrant City: Starnberg
Registrant State/Province: Bayern
Registrant Postal Code: 82319
Registrant Country: DE
Registrant Phone: +49.8151368670
Registrant Phone Ext:
Registrant Fax: +49.81513686777
Registrant Fax Ext:
Registrant Email: hostmaster@united-domains.de
Registry Admin ID:
Admin Name: Hostmaster Hostmaster
Admin Organization: united-domains AG
Admin Street: Gautinger Strasse 10
Admin City: Starnberg
Admin State/Province: DE
Admin Postal Code: 82319
Admin Country: DE
Admin Phone: +49.8151368670
Admin Phone Ext:
Admin Fax: +49.81513686777
Admin Fax Ext:
Admin Email: hostmaster@united-domains.de
Registry Tech ID:
Tech Name: Host Master
Tech Organization: united-domains AG
Tech Street: Gautinger Str. 10
Tech City: Starnberg
Tech State/Province: Bayern
Tech Postal Code: 82319
Tech Country: DE
Tech Phone: +49.8151368670
Tech Phone Ext:
Tech Fax: +49.81513686777
Tech Fax Ext:
Tech Email: hostmaster@united-domains.de
Name Server: ns1.udagdns.com
Name Server: ns2.udagdns.com
Name Server: ns3.udagdns.com
DNSSEC: 
URL of the ICANN WHOIS Data Problem Reporting System: http://wdprs.internic.net/
>>> Last update of WHOIS database: 2013-02-12T08:40:19Z

# Terms and conditions:
# 
# This data is provided by united-domains AG
# for information purposes, and to assist persons obtaining information
# about or related to domain name registration records.
# united-domains AG does not guarantee its accuracy.
# By submitting a WHOIS query, you agree that you will use this data
# only for lawful purposes and that, under no circumstances, you will
# use this data to
# 1) allow, enable, or otherwise support the transmission of mass
# unsolicited, commercial advertising or solicitations via E-mail
# (spam); or
# 2) enable high volume, automated, electronic processes that apply
# to this WHOIS server.
# These terms may be changed without prior notice.
# By submitting this query, you agree to abide by this policy.

