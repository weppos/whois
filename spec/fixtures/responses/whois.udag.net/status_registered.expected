#domain
  %s == "udag.net"

#domain_id
  %s == "59973274_DOMAIN_NET-VRSN"


#status
  %s == :registered

#available?
  %s be_falsey

#registered?
  %s be_truthy


#created_on
  %s %CLASS{time}
  %s %TIME{2001-02-11T20:15:57Z}

#updated_on
  %s %CLASS{time}
  %s %TIME{2013-02-12T08:40:19Z}

#expires_on
  %s %CLASS{time}
  %s %TIME{2014-02-11T20:15:57Z}


#registrar
  %s %CLASS{registrar}
  %s.id           == "1408"
  %s.name         == "united domains AG"
  %s.organization == "united domains AG"
  %s.url          == "http://www.united-domains.de/"


#registrant_contacts
  %s %CLASS{array}
  %s.size eq(1)
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  %s[0].id            == nil
  %s[0].name          == "Host Master"
  %s[0].organization  == "united-domains AG"
  %s[0].address       == "Gautinger Str. 10"
  %s[0].city          == "Starnberg"
  %s[0].zip           == "82319"
  %s[0].state         == "Bayern"
  %s[0].country       == nil
  %s[0].country_code  == "DE"
  %s[0].phone         == "+49.8151368670"
  %s[0].fax           == "+49.81513686777"
  %s[0].email         == "hostmaster@united-domains.de"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#admin_contacts
  %s %CLASS{array}
  %s.size eq(1)
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == nil
  %s[0].name          == "Hostmaster Hostmaster"
  %s[0].organization  == "united-domains AG"
  %s[0].address       == "Gautinger Strasse 10"
  %s[0].city          == "Starnberg"
  %s[0].zip           == "82319"
  %s[0].state         == "DE"
  %s[0].country       == nil
  %s[0].country_code  == "DE"
  %s[0].phone         == "+49.8151368670"
  %s[0].fax           == "+49.81513686777"
  %s[0].email         == "hostmaster@united-domains.de"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#technical_contacts
  %s %CLASS{array}
  %s.size eq(1)
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_TECHNICAL
  %s[0].id            == nil
  %s[0].name          == "Host Master"
  %s[0].organization  == "united-domains AG"
  %s[0].address       == "Gautinger Str. 10"
  %s[0].city          == "Starnberg"
  %s[0].zip           == "82319"
  %s[0].state         == "Bayern"
  %s[0].country       == nil
  %s[0].country_code  == "DE"
  %s[0].phone         == "+49.8151368670"
  %s[0].fax           == "+49.81513686777"
  %s[0].email         == "hostmaster@united-domains.de"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil


#nameservers
  %s %CLASS{array}
  %s.size eq(3)
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.udagdns.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.udagdns.com"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns3.udagdns.com"
